composer.json

```diff
@@ -43,8 +43,8 @@
         "psr/simple-cache-implementation": "~1.0"
     },
     "require-dev": {
-        "sabre/cs": "~1.0.0",
-        "phpunit/phpunit" : ">=6.2.4"
+        "friendsofphp/php-cs-fixer": "^2",
```

Comment:
I have bad experiences with this tool regarding BC - therefore I would recommend a non semver version constraint for it, like 2.16.* or similar

Produced codestyles highly depend on the minor version

Reply:
Yes, there are sometimes new code style checks/fixers that are release as a minor version, but actually they find new stuff in existing code and so break CI.
Changed it.